addons/payment_stripe/controllers/main.py

```diff
@@ -15,7 +15,11 @@
 from odoo.addons.payment import utils as payment_utils
 from odoo.addons.payment.logging import get_payment_logger
 from odoo.addons.payment_stripe import utils as stripe_utils
-from odoo.addons.payment_stripe.const import HANDLED_WEBHOOK_EVENTS, SENSITIVE_KEYS
+from odoo.addons.payment_stripe.const import (
```

Comment:
Instead you can simply do:

```suggestion
from odoo.addons.payment_stripe import const
```
and adapt `const.*` usages.